README.md

```diff
@@ -246,6 +246,37 @@ class ArticleBody < String
   end
 end
 ```
+
+## Documentation
+### Form
```

Comment:
I don't like that attribute sets and attributes fetches. Don't know if there is better wording to be found here